### DISCUSSION TITLE
how many active TX-RX can run simultaneously

### DISCUSSION THREAD

**Discussion by koolakoff** [ExpressLRS/ExpressLRS#2119] [Q&A] [answered]
Good day, team!

Please advice how many sets of ExpressLRS TX-RX may run simultaneously without interfering each other?
i.e. if we go race with 8 competitors, can they all use elrs and be sure that control will not be broken?

Or advice if there is any documentation on ELRS collisions/interfering handling to read?

Thanks in advance,
Peger

**Comment by JyeSmith** [comment, marked as answer]
You will get packet loss due to interference as soon as more than 1 tx is on.  But fail safe wont happen until you have in excess of a dozen(s).  However, if you use different RF modes (different packet rates) there will be less interference due to the modulation parameters.

The short answer is you will need far more transmitters on than video allows.  Good luck.

**Reply by koolakoff** [reply]
Thanks!
as I understood it uses some FHSS so actually different tx-rx pairs may use some differ frequency ranges to reduce interference?

**Reply by SunjunKim** [reply]
For example, in ELRS 2.4g link, there are 80 channels and assume there are N ELRS racers.
In theory, the chance of one packet getting lost is 2*(N-1)/80. 

Therefore if you get 100 LQ when you fly solo. In the worst case, you'll get approximately 82 LQ when you fly within a pack of 8 racers. 82 LQ is very far from failsafe per se, unless other strong interference gets involved.

**Reply by koolakoff** [reply]
Got it! 
So from this formula 41st pilot will make everyone get 100% lose, or 21st will make 50% - am I correct?

**Reply by koolakoff** [reply]
Is there any documentation describing this phisics? I.e the actual freq limits,  channel numbers, the formula from above?.. or the packets sizes and structure?.. (yeah, packets details are out if this topic scope, but also unteresting topic...)
